cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -0,0 +1,429 @@
+      inside:    _container
+    } );
+
+    _header = $.create( 'header', {
+      className: 'cf-multi-select_header'
+    } );
+
+    _search = $.create( 'input', {
+      className:   'cf-multi-select_search',
+      type:        'text',
+      placeholder: _placeholder || 'Choose up to five',
+      inside:      _header
+    } );
+
+    _fieldset = $.create( 'fieldset', {
+      className:  'cf-multi-select_fieldset',
+      visibility: 'hidden'
+    } );
+
+    _list = $.create( 'ul', {
+      className: 'list__unstyled cf-multi-select_options',
+      inside:    _fieldset
+    } );
+
+    _optionData.forEach( function( option ) {
+      var li = $.create( 'li', {
+        'data-option': option.value
+      } );
+
+      $.create( 'input', {
+        id:        option.value,
+        value:     option.value,
+        type:      'checkbox',
+        name:      _name,
+        class:     'cf-input cf-multi-select_checkbox',
+        inside:    li
+      } );
+
+      $.create( 'label', {
+        for:       option.value,
+        innerText: option.text,
+        className: 'cf-multi-select_label',
+        inside:    li
+      } );
+
+      _list.appendChild( li );
+    } );
+
+    // Write our new markup to the DOM
+    _container.appendChild( _header );
+    _container.appendChild( _fieldset );
+  };
+
+  function _highlight( direction ) {
+    var count = _filtered.length;
+    var value;
+    var item;
+
+    if ( direction === DIR_NEXT && _index < count - 1 ) {
+      _index += 1;
+    } else if ( direction === DIR_PREV && _index > -1 ) {
+      _index -= 1;
+    }
+
+    if ( _index > -1 ) {
+      var value = _filtered[ _index ].value;
+      var item = _list.querySelector( '[data-option="' + value + '"]' );
+      var input = item.querySelector( 'input' );
+
+      _isBlurSkipped = true;
+      input.focus();
+    } else {
+      _isBlurSkipped = false;
+      _search.focus();
+    }
+  };
+
+  function _updateSelections( value ) {
+    var option = arrayHelpers.valInArray( _optionData, value ).item ||
+                 _optionData[_index];
+
+    if ( option ) {
+      var inSelections = arrayHelpers.valInArray( _selections, option.value );
+      var li;
+
+      if ( inSelections ) {
+        li = _choices.querySelector( 'li[data-option="' + option.value + '"]' );
+
+        _selections.splice( inSelections.index, 1 );
+
+        if ( li ) {
+          _choices.removeChild( li );
+        }
+      } else {
+        li = $.create( 'li', {
+          'data-option': option.value
+        } );
+
+        $.create( 'label', {
+          'for': option.value,
+          'innerHTML': option.text,
+          inside: li
+        } );
+
+        _choices.appendChild( li );
+        _selections.push( option );
+      }
+    }
+
+    _index = -1;
+    _isBlurSkipped = false;
+
+    if ( _fieldset.getAttribute( 'visibility' ) === 'visible' ) {
+      _search.focus();
+    }
+  };
+
+  function _evaluate() {
```

Comment:
Consider passing in `_search.value` into `_evaluate()` so the method signature looks like `_evaluate( value )`.